### PR TITLE
add demo page

### LINKED ISSUE #1066: Demo page
I think that we definitely need to have a demo page which contains some examples all of the blocks in a current state. Of course this page should be automatically updated on each commit to the main branch.
What's reason we don't have it yet? It seems that's a simple task via github gh-pages.

@veged @narqo @tadatuta @mishanga 

?

### PR DESCRIPTION
**Pull request by sipayRT**
closes #1066

### CODE REVIEW

**Comment by tadatuta** [comment]
I think we need two languages or at least use English as default.

File: desktop.pages/showcase/showcase.bemjson.js [hunk 1]
```diff
@@ -0,0 +1,193 @@
+({
+    block : 'page',
+    title : 'Showcase',
+    mods : { theme : 'islands' },
+    head : [
+        { elem : 'css', url : '_showcase.css' },
+        { elem : 'js', url : '_showcase.js' }
```
Comment (tadatuta) [inline desktop.pages/showcase/showcase.bemjson.js]:
``` js
scripts: [
    { elem : 'js', url : '_showcase.js' }
]
```

**Comment by sipayRT** [comment]
change language to English

**Comment by tadatuta** [comment]
fine for me.
@dfilatov @veged, can we merge?

**Comment by hcodes** [comment]
4 radio buttons:
- Radio is off
- Radio is on
- Disabled radio is off
- Disabled radio is on 

4 checkboxes:
- Unchecked
- Checked
- Disabled unchecked
- Disabled checked 

As a example - http://designmodo.github.io/Flat-UI/

**Comment by hcodes** [comment]
Spin should not be in the button...

**Comment by sipayRT** [comment]
why spin shouldn't be in the button?

**Comment by sipayRT** [comment]
@hcodes

**Comment by hcodes** [comment]
https://github.com/bem/bem-components/issues/1065

**Comment by sipayRT** [comment]
tnx for a clarification. I removed this example

**Comment by sipayRT** [comment]
ping!

**Pull request merged**